src/trees/my-issues-tree.ts

```diff
@@ -14,6 +14,8 @@ export class MyIssuesTree implements vscode.TreeDataProvider<Issue> {
       key: config.apiKey,
       additionalHeaders: config.additionalHeaders,
       rejectUnauthorized: config.rejectUnauthorized,
+      filter: config.filter,
+      sort: config.sort
```

Comment:
Missing `filter: config.filter,` and `sort: config.sort` in `projects-tree.ts` and `extention.ts`